Action fails after the recent release

```plain
##[debug]Evaluating condition for step: 'Set up OCaml'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Set up OCaml
##[debug]Loading inputs
##[debug]Evaluating: env.OPAM_COMP
##[debug]Evaluating Index:
##[debug]..Evaluating env:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'OPAM_COMP'
##[debug]=> '4.11.1'
##[debug]Result: '4.11.1'
##[debug]Loading env
Run avsm/setup-ocaml@v1
  with:
    ocaml-version: 4.11.1
  env:
    HEAD_REF: ci572279844
    RUN_URL: https://github.com/canonical/multipass-private/actions/runs/572279844
    BUILD_DIR: ../build
    MACOSX_DEPLOYMENT_TARGET: 10.14
    OPAM_COMP: 4.11.1
    OPAMVERBOSE: 1
    OPAMYES: 1
    HOMEBREW_NO_AUTO_UPDATE: 1
    QT_VERSION: 5.12.10
    PKG_CONFIG_PATH: /Users/cibot/actions-runner/_work/multipass-private/Qt/5.12.10/clang_64/lib/pkgconfig
    Qt5_Dir: /Users/cibot/actions-runner/_work/multipass-private/Qt/5.12.10/clang_64
    Qt5_DIR: /Users/cibot/actions-runner/_work/multipass-private/Qt/5.12.10/clang_64
    QT_PLUGIN_PATH: /Users/cibot/actions-runner/_work/multipass-private/Qt/5.12.10/clang_64/plugins
    QML2_IMPORT_PATH: /Users/cibot/actions-runner/_work/multipass-private/Qt/5.12.10/clang_64/qml
/usr/local/bin/brew install opam
Warning: opam 2.0.8 is already installed and up-to-date.
To reinstall 2.0.8, run:
  brew reinstall opam
/usr/local/bin/opam init --bare -yav https://github.com/ocaml/opam-repository.git

User configuration:
  ~/.bash_profile is already up-to-date.
[NOTE] Make sure that ~/.bash_profile is well sourced in your ~/.bashrc.

/Users/cibot/actions-runner/_work/_actions/avsm/setup-ocaml/v1/dist/install-ocaml-unix.sh 4.11.1
++ opam info -f version ocaml --color=never
+ CURRENT_OCAML=4.11.1
+ '[' 4.11.1 '!=' 4.11.1 ']'
+ opam switch set 4.11.1
Error: Error: The process '/Users/cibot/actions-runner/_work/_actions/avsm/setup-ocaml/v1/dist/install-ocaml-unix.sh' failed with exit code 5
##[debug]Node Action run completed with exit code 1
##[debug]OPAMJOBS='4'
##[debug]OPAMYES='1'
##[debug]Finishing: Set up OCaml
```

On the host:
```plain
$ opam switch set 4.11.1
[ERROR] No switch 4.11.1 is currently installed. Did you mean 'opam switch create 4.11.1'?
        Installed switches are:
          - default
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [unreleased]
 
+## Fixed
+
+- Further fix to switch initialisation
+
 ## [1.1.8]
 
 ### Changed

--- a/src/install-ocaml-unix.sh
+++ b/src/install-ocaml-unix.sh
@@ -2,14 +2,4 @@
 
 set -ex
 
-# opam has been initialised, but we need to switch to
-# the right version from the system compiler
-
-CURRENT_OCAML=$(opam info -f version ocaml --color=never)
-
-if [ "$CURRENT_OCAML" != "$1" ]; then
-  opam switch set "$1" 2>/dev/null || opam switch create "$1" "$1"
-else
-  # opam has been initialized with --bare, the switch needs to be activated
-  opam switch set "$1" 2>/dev/null
-fi
+opam switch set "$1" 2>/dev/null || opam switch create "$1" "$1"

